web-app/src/components/atoms/Input.vue

```diff
@@ -1,11 +1,23 @@
 <script setup lang="ts">
-  const props = defineProps(['id', 'type', 'placeholder']);
-  const inputModel = defineModel('inputModel');
-  const computedType = props.type || 'text';
+  import { ref, computed, watch } from 'vue';
+
+  const emit = defineEmits(['update:value']);
+  const props = defineProps(['value', 'id', 'type', 'placeholder']);
+
+  //reactive states
+  const localValue = ref(props.value);
+  const computedType = computed(() => {
+    return props.type || 'text';
+  });
```

Comment:
Something to fix:
Props can be defined using object syntax, and default value can be passed to "type"

Reply:
Fixed